cxx-qt-gen/src/writer/cpp/header.rs

```diff
@@ -71,6 +71,8 @@ pub fn write_cpp_header(generated: &GeneratedCppBlocks) -> String {
           {members}
         }};
 
+        static_assert(std::is_base_of<QObject, {ident}>::value, "QObject must be the base of the class");
```

Comment:
```suggestion
        static_assert(std::is_base_of<QObject, {ident}>::value, "{ident} must inherit from QObject");
```
I think this wording makes it a bit clearer that the class doesn't need to *directly* inherit from QObject.

Reply:
Done